components/countdown.js

```diff
@@ -43,7 +43,7 @@ export function CompactLongCountdown (props) {
                 ? ` ${props.formatted.hours}:${props.formatted.minutes}:${props.formatted.seconds}`
                 : Number(props.formatted.minutes) > 0
                   ? ` ${props.formatted.minutes}:${props.formatted.seconds}`
-                  : Number(props.formatted.seconds) > 0
+                  : Number(props.formatted.seconds) >= 0
```

Comment:
Since I removed the state update after the countdown runs on, I am using `>=` here now so the timer doesn't disappear when timer ran out but the next poll is pending.

Maybe I shouldn't have removed the state update but this makes the code simpler (less state) + it exposes the rare bug by getting stuck at 0 seconds.